pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -239,9 +239,10 @@ def make_package(args):
     assets_dir = "src/main/assets"
 
     # Delete the old assets.
-    try_unlink(join(assets_dir, 'public.mp3'))
-    try_unlink(join(assets_dir, 'private.mp3'))
+    if os.path.exists(assets_dir):
+        shutil.rmtree(assets_dir)
```

Comment:
I'm allergic to `if` :smile: 
```suggestion
    shutil.rmtree(assets_dir, ignore_errors=True)
```